bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/ESP32ClangToolChain.java

```diff
@@ -21,7 +22,13 @@ public ESP32ClangToolChain(IToolChainProvider provider, Path pathToToolChain)
 		setProperty("ATTR_ID", ESP32ClangCmakeToolChainProvider.TOOLCHAIN_NAME); //$NON-NLS-1$
 	}
 
-	@Override
+	// API after Changes in CDT 10.5.0
+	public List<String> getBinaryParserIds()
+	{
+		return List.<String>of("org.eclipse.cdt.core.ELF"); //$NON-NLS-1$
+	}
```

Comment:
Can we use `return List.<String>of(CCorePlugin.PLUGIN_ID + ".ELF")`

Reply:
Also, I see this is implemented only in ESP32ClangToolChain, how about other toolchain classes? Do they have any impact
https://github.com/espressif/idf-eclipse-plugin/blob/0448d22c539249bbaac8cf3cb8eb61bf66bf62f4/bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/AbstractESPToolchain.java#L29

Reply:
Hi @kolipakakondal, thanks for the review. add CCorePlugin.PLUGIN_ID + ".ELF in the latest commit. 
We are using getBinaryParserIds() also in AbstactESPToolchain class, which is a parent for other toolchains